core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -4652,6 +4652,12 @@ object ZIO extends ZIOCompanionPlatformSpecific {
   def logInfo(message: => String)(implicit trace: ZTraceElement): UIO[Unit] =
     new Logged(ZLogger.stringTag, () => message, someInfo, trace = trace)
 
+  /**
+   * Sets the log level for this effect.
+   */
+  def logLevel(level: LogLevel): LogLevel =
```

Comment:
hmm.. this looks wrong

Reply:
No `logLevel` has an `apply` method so you use it like `logLevel(LogLevel.Fine)(zio)`.

Reply:
could you please add this example to the scaladoc above? I wouldn't have guessed that from the signature alone

Reply:
It is already in the documentation.

Reply:
I meant like 
```scala

/**
   * Sets the log level for this effect.
   *
   * Use this to wrap your effect like 
   * <code>
   * ZIO.logLevel(LogLevel.Fine)(myEffect)
   * </code>
   */
  def logLevel(level: LogLevel): LogLevel =
   level
   ```

Reply:
Oh sure. Added now.